src/Game.qml

```diff
@@ -1,6 +1,8 @@
 import QtQuick 2.9
 import Nemo.Configuration 1.0
 import org.nemomobile.lipstick 0.1
+import org.asteroid.utils 1.0
+import org.asteroid.controls 1.0
```

Comment:
Why do these become necessary now ?

Reply:
`org.asteroid.utils` is needed to check if the display is round.
If the display is round, show a round animation of the game over overlay.
Else, it's a square overlay.